lib/rubocop/rspec/cop_helper.rb

```diff
@@ -1,102 +0,0 @@
-# frozen_string_literal: true
-
-require 'tempfile'
```

Comment:
Is this file no longer needed ?

Reply:
Yep, they made the spec helper accessible! https://github.com/puppetlabs/rubocop-i18n/pull/34/commits/c5296d85f380e4ef9f6f0fd5710f653879178421